docs/setup.md

```diff
@@ -0,0 +1,88 @@
+grep -qxF 'composer.lock' .git/info/exclude || echo 'composer.lock' >> .git/info/exclude
+```
+
+The Pantheon Lando recipe can connect the local development environment to a remotely hosted site. Connection information is stored in a local Lando settings file and includes a Pantheon site name and UUID. While we can connect to any site running this upstream, a particular integration environment has been provisioned for engineers working on this repository or the associated YaleSites profile. Connection information for the integration environment is stored in the example Lando local file.
+
+```bash
+# Step 3: Create a local Lando settings file to connect to the external environment.
+cp .lando.local.example.yml .lando.local.yml
+```
+
+Starting Lando will provision the containers required to run a local development environment. The `pull` command can then sync the database and files from a Pantheon-hosted site with the new local site.
+
+```bash
+# Step 4: Start Lando and import the remote files and database.
+lando start
+lando pull --database=dev --files=dev --code=none
+```
+
+Visit the local dev site [https://yalesites-project.lndo.site/](https://yalesites-project.lndo.site/) or run `lando drush uli` to obtain a login link.
+
+## Working on the YaleSites Profile
+
+The [YaleSites installation profile](https://github.com/yalesites-org/yalesites_profile) combines a suite of modules, themes, pre-defined configurations, and custom code into a single installable package. The majority of development on YaleSite's platform will take place within this profile. The previously created local development environment is an ideal place for working on the profile within the context of a YaleSite.
+
+By default, composer dependencies are downloaded in a dist packaged version of the project with git metadata removed. When working on the `yalesites_profile` package, the originally downloaded composer dependency must be replaced with the source packaged version. This allows changes to the profile repo to be tracked in version control.
+
+```bash
+# Step 1: Configure Composer to use source packaged versions.
+lando composer config --global 'preferred-install.yalesites-org/*' source
+
+# Step 2: Manually remove the originally downloaded dist packed version.
+sudo rm -R web/profiles/contrib/yalesites_profile
+
+# Step 3: Use Composer to download the new version of the profile.
+lando composer update yalesites_profile
+
+# Step 4: Verify that the profile is tracking a remote repository.
+git -C web/profiles/contrib/yalesites_profile ls-remote --get-url
+# Returns: https://github.com/yalesites-org/yalesites_profile.git
+```
+
+Alternatively, developers can use a symbolic link or Composer local repositories to allow the profile to be tracked within the `yalesites-project`. Each option has pros and cons. Any solution using sub-modules should be avoided as Composer can not support this workflow in a meaningful way.
```

Comment:
The steps in this section work without error, but I'm still unclear how you to contribute changes to the yalesites_profile. After finishing these docs, I have a working local site in Lando and now I want to add a module to the yalesites_profile, update and export config, etc. How do I do that?

Reply:
I would like to keep this file strictly to localdev setup. What do you think about a separate 'development' document or something of that form? It can include examples and help:
- Useful drush and terminus commands
- Working in the profile: Add a module, update config
- Working in a theme

Reply:
Yes, 100%!